.gitignore

```diff
@@ -161,4 +161,8 @@ cython_debug/
 # conda environment
 environment.yml
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+!.vscode/settings.json
```

Comment:
Allows commiting vscode files needed without git complaint

Reply:
In reaction to a request from Tom.  The `settings.json` file has been committed for ages, and contains a standard for formatting the files on save etc.  It's already there on `master` - this just stops git from complaining that it's in a commit.

Currently it's a tracked file that is _also_ in the `.gitignore`

Reply:
RE: the `recommendations.json` file allows us to make suggestions for useful plugins for working with the codebase, but honestly, I don't feel strongly about whether we have it or not.